src/core/Button/Button.md

```diff
@@ -1,11 +1,18 @@
 ```js
 import { Button } from 'suomifi-ui-components';
+import React from 'react';
+
+const testi = React.createRef();
 
 <>
   <Button
     className="my-button--test"
     onClick={() => console.log('Test button click')}
     type="submit"
+    ref={testi}
```

Comment:
should this be in english `test` rather than `testi`?

Reply:
Ah, this is something I meant to do before pressing 'ready for review' but forgot 😅 Many components still have these makeshift test examples. Do you think it would be necessary to present the ref prop in the examples in the first place? I originally thought about simply getting rid of them when I'm done.

Reply:
Not necessary, but might be beneficial also as I think this is pretty commonly used pattern. Could drop for buttons, but leave in place for inputs?

Reply:
Looks good now!